lib/cli/ui/progress.rb

```diff
@@ -59,6 +59,8 @@ def initialize(width: Terminal.width)
       # * +:percent+ - Increment progress by a specific percent amount
       # * +:set_percent+ - Set progress to a specific percent
       #
+      # *Obs:* The +:percent+ and +:set_percent must be between 0.00 and 1.0
```

Comment:
Is `Obs` a typo? I'm not sure what you're going for there.

Reply:
What I mean here is to add a reminder like an observation (Obs) 😅 Do you suggest a better approach?

Reply:
Oh, I see, I'd go with
```suggestion
      # *Note:* The +:percent+ and +:set_percent must be between 0.00 and 1.0
```